### PR TITLE
feat(Request Data Collector): Add RequestResponseCollector and DatabaseCollector

### PR DESCRIPTION
**Pull request by donatorsky**
Add RequestResponseCollector and DatabaseCollector.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+
+[VERSION]
```
Comment (maksymilian-lewicki) [inline .editorconfig]: We don't have `VERSION` file.

File: README.md [hunk 2]
```diff
@@ -1 +1,178 @@
...
+
+When using this package, every handled request (i.e. not excluded according to `exclusions` rules) will contain `X-Request-Id` header in the response. You can use this ID to correlate various collectors' data.
+
+## Set up
+Firstly, register package's service provider in `config/app.php` file:
+
+```php
+	'providers' => [
+		// ...
+		\Pitched\RequestDataCollector\RequestDataCollectorServiceProvider,
```
Comment (maksymilian-lewicki) [inline README.md]: rename `Pitched` to `Miquido`

File: config/request-data-collector.php [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    |--------------------------------------------------------------------------
+    | Exclude request
+    |--------------------------------------------------------------------------
+    |
+    | Exclude requests matching to listed filters.
+    |
+    */
+    'exclude' => [
+        [
+            'filter' => \Pitched\RequestDataCollector\Filters\UserAgentFilter::class,
```
Comment (maksymilian-lewicki) [inline config/request-data-collector.php]: rename `Pitched` to `Miquido`

**Review by maksymilian-lewicki — CHANGES_REQUESTED** [review]
Rename namespaces to Miquido in all files

**Review by miquido-legenc — APPROVED** [review]

**Review by maksymilian-lewicki — APPROVED** [review]

**Pull request merged**